examples/step-58/doc/results.dox

```diff
@@ -0,0 +1,207 @@
+<h1>Results</h1>
+
+Running the code results in screen output like the following:
+```
+Number of active cells: 4096
+Number of degrees of freedom: 16641
+
+Time step 1 at t=0
+Time step 2 at t=0.00390625
+Time step 3 at t=0.0078125
+Time step 4 at t=0.0117188
+[...]
+```
+Running the program also yields a good number of output files that we will
+visualize in the following.
+
+
+<h3>Visualizing the solution</h3>
+
+The `output_results()` function of this program generates output files that
+consist of a number of variables: The solution (split into its real and imaginary
+parts), the magnitude, and the phase. If we visualize these four fields, we get
+images like the following after a few time steps (at time $t=0.242$, to be
+precise:
+
+<div class="twocolumn" style="width: 80%">
+  <div>
+    <img src="https://www.dealii.org/images/steps/developer/step-58.re.png"
+         alt="Real part of the solution at t=0.242"
+         width="400">
+  </div>
+  <div>
+    <img src="https://www.dealii.org/images/steps/developer/step-58.im.png"
+         alt="Imaginary part of the solution at t=0.242"
+         width="400">
+  </div>
+  <div>
+    <img src="https://www.dealii.org/images/steps/developer/step-58.magnitude.png"
+         alt="Magnitude of the solution at t=0.242"
+         width="400">
+  </div>
+  <div>
+    <img src="https://www.dealii.org/images/steps/developer/step-58.phase.png"
+         alt="Phase of the solution at t=0.242"
```

Comment:
Is there a reason to have a grid in this plot? I would remove the grid

Reply:
Yes, that's actually an important part because I want to show that the phase changes from +close to pi to -close to pi over the course of just one cell. It refers to the discussion below.